sys/include/senml.h

```diff
@@ -0,0 +1,158 @@
+#define SJ_BASE_TIME     "bt"   /**< Key for the base time attribute in JSON documents     */
+#define SJ_BASE_UNIT     "bu"   /**< Key for the base unit attribute in JSON documents     */
+#define SJ_BASE_VALUE    "bv"   /**< Key for the base value attribute in JSON documents    */
+#define SJ_BASE_SUM      "bs"   /**< Key for the base sum attribute in JSON documents      */
+#define SJ_NAME          "n"    /**< Key for the name attribute in JSON documents          */
+#define SJ_UNIT          "u"    /**< Key for the unit attribute in JSON documents          */
+#define SJ_VALUE         "v"    /**< Key for the value attribute in JSON documents         */
+#define SJ_STRING_VALUE  "vs"   /**< Key for the string value attribute in JSON documents  */
+#define SJ_BOOL_VALUE    "vb"   /**< Key for the boolean value attribute in JSON documents */
+#define SJ_VALUE_SUM     "s"    /**< Key for the value sum attribute in JSON documents     */
+#define SJ_TIME          "t"    /**< Key for the time attribute in JSON documents          */
+#define SJ_UPDATE_TIME   "ut"   /**< Key for the update time attribute in JSON documents   */
+#define SJ_DATA_VALUE    "vd"   /**< Key for the data value attribute in JSON documents    */
+#define SJ_LINK          "l"    /**< Key for the link attribute in JSON documents          */
+
+#define SC_VERSION       (-1)   /**< Key for the version attribute in CBOR documents       */
+#define SC_BASE_NAME     (-2)   /**< Key for the base name attribute in CBOR documents     */
+#define SC_BASE_TIME     (-3)   /**< Key for the base time attribute in CBOR documents     */
+#define SC_BASE_UNIT     (-4)   /**< Key for the base unit attribute in CBOR documents     */
+#define SC_BASE_VALUE    (-5)   /**< Key for the base value attribute in CBOR documents    */
+#define SC_BASE_SUM      (-6)   /**< Key for the base sum attribute in CBOR documents      */
+#define SC_NAME          (0)    /**< Key for the name attribute in CBOR documents          */
+#define SC_UNIT          (1)    /**< Key for the unit attribute in CBOR documents          */
+#define SC_VALUE         (2)    /**< Key for the value attribute in CBOR documents         */
+#define SC_STRING_VALUE  (3)    /**< Key for the string value attribute in CBOR documents  */
+#define SC_BOOL_VALUE    (4)    /**< Key for the boolean value attribute in CBOR documents */
+#define SC_VALUE_SUM     (5)    /**< Key for the value sum attribute in CBOR documents     */
+#define SC_TIME          (6)    /**< Key for the time attribute in CBOR documents          */
+#define SC_UPDATE_TIME   (7)    /**< Key for the update time attribute in CBOR documents   */
+#define SC_DATA_VALUE    (8)    /**< Key for the data value attribute in CBOR documents    */
+#define SC_LINK          (9)    /**< Key for the link attribute in CBOR documents          */
+
+/**
+ * @brief The different data types SenML supports
```

Comment:
Check for this file: sometime you start the comments with a capital letter, sometimes with small letter.

You might also want to put 3 space chars after the @brief (tab alignment) - this is the default in RIOT...